src/main/resources/application.yml

```diff
@@ -51,6 +51,10 @@ spring:
     initializer:
       enabled: false
     enable-auto-commit: true
+  async:
+    config:
+      executor:
+        pool-size: ${SCHEDULAR_POOL_SIZE:100}
```

Comment:
Spelling error "SCHEDULAR" for the new env variable.

Reply:
corrected